analysis/generate_report.py

```diff
@@ -81,6 +83,11 @@ def get_arg_parser():
                         action='store_true',
                         default=False,
                         help='If set, clang coverage reports are linked.')
+    parser.add_argument('-diff',
+                        '--differential-graphs',
+                        action='store_true',
+                        default=False,
+                        help='If set, differential graphs are shown.')
```

Comment:
Do we need a both --diff and --cov? How about having something like a single `--detailed-coverage-report` flag that adds both the HTML source coverage reports and the plots too?

Reply:
Also, why should it default to False?